server/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
-name = "server-actix"
+name = "server"
 version = "0.1.0"
-authors = ["loopy"]
```

Comment:
@VictorXLR were you signing you name "loopy"?

Reply:
No, My system user name is `loopy`, Ill change that to VictorXLR now